.github/workflows/ci.yml

```diff
@@ -52,11 +54,28 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v4
 
-    - name: Get commit count
-      id: get_commit_count
+    # - name: Get commit count
```

Comment:
@marshal-dteach Could you please remove the redundant code comments?

Reply:
Will do